packages/now-next/src/index.ts

```diff
@@ -204,7 +205,30 @@ export const build = async ({
 }> => {
   validateEntrypoint(entrypoint);
 
-  const isSharedLambdas = config.sharedLambdas;
+  const nowJsonPath = await findUp(['now.json', 'vercel.json'], {
+    type: 'file',
+  });
+  let hasLegacyRoutes = false;
+
+  if (nowJsonPath) {
+    const nowJsonData = JSON.parse(await readFile(nowJsonPath, 'utf8'));
+
+    if (Array.isArray(nowJsonData.routes) && nowJsonData.routes.length > 0) {
+      hasLegacyRoutes = true;
+      console.warn(
+        `WARNING: your application is being opted out of @vercel/next's optimized lambdas mode due to legacy routes in ${path.basename(
+          nowJsonPath
+        )}. http://err.sh/vercel/vercel/now-next-legacy-routes-optimized-lambdas`
```

Comment:
Update link here as well:

```suggestion
        )}. http://err.sh/vercel/vercel/next-legacy-routes-optimized-lambdas`
```